pkgs/modules/python/default.nix

```diff
@@ -98,7 +98,7 @@ let
     inherit pkgs python pypkgs debugpy;
   };
 
-  debuggerConfig = if pythonVersion == "3.8" then ({ }) else
+  debuggerConfig = if pythonVersion == "3.13" then ({ }) else
```

Comment:
no need to check for 3.13 - we'll take care of that part

Reply:
isn't it best to check for 3.13 from now, for future-proofing? 3.13 isn't supported by `debugpy` but 3.12 is (to practically the same extent as 3.11): https://github.com/microsoft/debugpy/pull/1359

Reply:
we don't support 3.13 yet - when we do, 3.13 might be supported by debugpy already